src/components/HorizontalList/HorizontalListImage/HorizontalListImage.tsx

```diff
@@ -0,0 +1,119 @@
+import { HorizontalList } from "../HorizontalList";
+import { useEffect, useState } from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import HorizontalImage from "../../../images/HorizontalImage.svg";
+import "swiper/css";
+interface AboutItem {
+  src?: string; // Optional since the fallback may not have an image
+  alt?: string;
+  title: JSX.Element | string;
+  description: JSX.Element | string;
+}
+export function HorizontalListImage() {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+  const isMobile = windowWidth <= 850;
+
+  const aboutItems: AboutItem[] = [
+    {
+      src: HorizontalImage,
+      alt: "Don",
+      title: <>Don Stoner</>,
+      description: <>Co-Owner Expert Event Solutions LLC</>,
+    },
+    {
+      src: HorizontalImage,
+      alt: "Don",
+      title: <>Co-Owner</>,
+      description: <>Co-Owner Expert Event Solutions LLC</>,
+    },
+    {
+      src: HorizontalImage,
+      alt: "Don",
+      title: <>Manager</>,
+      description: <>Manager at Expert Event Solutions LLC</>,
+    },
+    {
+      src: HorizontalImage,
+      alt: "Don",
+      title: <>Manager 2</>,
+      description: <>Manager 2 Expert Event Solutions LLC</>,
+    },
+  ];
+
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setWindowWidth(width);
+    };
+
+    handleResize();
+
+    window.addEventListener("resize", handleResize);
+
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  return (
+    <div>
+      {!isMobile ? (
+        <HorizontalList
+          containerClassName="bg-wheat"
+          unorderedListClassName="flex gap-[86px] pt-[10px] justify-center"
+        >
+          {aboutItems.map((item, index) =>
+            item.src && item.alt ? ( // Ensure src and alt exist
```

Comment:
Having an `alt` on images is best practice, but if we don't have an `alt` we shouldn't not render the image.